projects/storefrontstyles/scss/layout/_index.scss

```diff
@@ -33,3 +33,8 @@ cx-page-layout {
 .LoginPageTemplate {
   @extend %LoginPageTemplate !optional;
 }
+
+.AccountPageTemplate {
+  --cx-max-width: 100%;
+  @extend %LoginPageTemplate !optional;
```

Comment:
I'm wondering if the LoginPageTemplate should extend the AccountPageTemplate?

Reply:
Or why `.AccountPageTemplate` class doesn't extend it's own `%AccountPageTemplate`?

Reply:
Yes, its a bit in between situation. Once all page templates are moved to this approach, i'm planning on eliminating backend templates. The front-end will probably lead, and AccountPageTemplate, LoginPageTemplate, NotFound, etc. are going to be merged. I need to come with a good name, and then it's time to move. I'm delaying this decision, first want to have good visibility on all of them.